shell/platform/android/android_context_gl.cc

```diff
@@ -105,10 +106,121 @@ static bool TeardownContext(EGLDisplay display, EGLContext context) {
   return true;
 }
 
+class AndroidEGLSurfaceDamage {
```

Comment:
Someone familiar with Android and EGL extensions should look at this.

Suggestions @chinmaygarde ?